services/QuillLMS/app/controllers/api/v1/app_settings_controller.rb

```diff
@@ -0,0 +1,17 @@
+class API::V1::AppSettingsController < ApplicationController
+  def index
+    app_settings = AppSetting.all
```

Comment:
I opted not to go with a UserAppSetting join table, since it's unnecessary overhead if user whitelists are reasonably short.